### PR TITLE
finish local test for SSSP

### PR DESCRIPTION
**Pull request by zpltys**

### CODE REVIEW

File: src/algorithm/SSSP.go [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+	p := old[n-1]
+	*pq = old[0 : n-1]
+	return p
+}
+
+
+// store the weight of edge connected to DstID
+type BoundMsg struct {
+	DstId graph.ID
+	RouteLen int64
```
Comment (bupt-wenxiaole) [inline src/algorithm/SSSP.go:45]: I think DstId means NodeID in FO, Routelen means distance in SSSP?  BUT graph.ID and RouteLen make me confused,

File: src/algorithm/SSSP.go [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+}
+
+// in the result -- routeTable, routeTable[i] stores a list of BoundMsg,
+// routeTable[node_i][j].RouteLen is the weight of edge: node_i -> routeTable[node_i][j].DstID
+func GenerateRouteTable(FO map[graph.ID][]graph.RouteMsg) map[graph.ID][]*BoundMsg {
```
Comment (bupt-wenxiaole) [inline src/algorithm/SSSP.go:50]: what's the differs between graph.RouteMsg with BoundMsg

**Review by bupt-wenxiaole — COMMENTED** [review]
Ok, I nearly understand your idea from your code, thanks for contribution, I will merge this PR to master branch

File: src/algorithm/SSSP.go (hunk 1 above)
Comment (zpltys) [reply src/algorithm/SSSP.go:45]: I will write more notes for it soon, and commit it in next pr

**Pull request merged**